### DISCUSSION TITLE
Proposal title

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1535] [Proposals]
### Goal

Como usuario de Mística me gustaría poder tener más opciones a la hora de crear un empty state.

### Context

Actualmente el componente _empty state card_ solo permite añadir un párrafo en el contenido y en la parte de acciones, solo poner un único link.

<img width="375" alt="Captura de pantalla 2023-10-26 a las 15 43 30" src="https://github.com/Telefonica/mistica-design/assets/129079621/f8611706-1ece-4751-943c-3365cf412448">

Para el caso que necesito solucionar, me gustaría poder incluir más de un párrafo en el contenido y añadir varios links en forma de listado.

<img width="376" alt="Captura de pantalla 2023-10-26 a las 15 47 15" src="https://github.com/Telefonica/mistica-design/assets/129079621/cd5f707f-c38f-4a94-b87d-6b53ea69b943">

### Impact

Se podría flexibilidad la creación de diferentes empty states en función del contexto.

### Discussion Questions

¿Se puede plantear esta flexibilización para modificar la parte del contenido y de las acciones?

### Do you have a visual proposal?

https://www.figma.com/file/TqkSvZMBiM9YLE4gZz7HWh/Search?type=design&node-id=1-4&mode=design

**Comment by aweell** [comment]
Hey @JoseTelefonica 

Una solución que se me ocurre para este caso (Teniendo en cuenta que el empty sea la solución) con respecto a los párrafos es soportar, como hacemos en el feedback, multiples párrafos en description pasándole un array.

En el caso de pasarle unos links en ese array también parece tragar...no se lo válido o no que será eso: https://tinyurl.com/yskxwynp

**Reply by yceballost** [reply]
me atrevería a decir que no es posible ese ejemplo que pones ahí, lo que admite el feedback es un array de strings. No sé si deberíamos añadir por un lado el multipárrafo y por otro un slot debajo del contenido?

**Reply by aweell** [reply]
Tiene todo el sentido que sea description array + slot, de hecho ahora mismo al feedback info le puedes pasar el extra. Quizás esto debería formar parte de ese alineamiento entre los feedbacks y los empties que se viene cociendo.

**Comment by aweell** [comment]
@JoseTelefonica,

Divido la solución en dos partes:

* **Multipárrafo:** Convertimos esta discussion en una issue para permitir pasarle multiples textos a la description tal y como funciona en el feedback.
* **Lista de links:** Estos links no tienen porqué formar parte del componente y se podrían introducir inmediatamente debajo como se muestra en este ejemplo -> https://tinyurl.com/yr5jztm9
